### DISCUSSION TITLE
Fields within actions don't respect field->required(true) validation rules

### DISCUSSION THREAD

**Discussion by squpshift** [laravel/nova-issues#5363] [Q&A] [answered]
Nova 4.22.1
Laravel 9.52.4

I am experiencing this exactly as described below on previously closed issue:

              `->rules('required')` works as expected for a `Select` field for me as of v4, however `->required()` only adds the `*` next to the field, it does not actually require it.

_Originally posted by @bkuhl in https://github.com/laravel/nova-issues/issues/2057#issuecomment-1374150818_

**Comment by crynobone** [comment]
We have dusk tests covering this usage and it doesn't failed right now.

**Comment by crynobone** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/172966/222028571-09d215eb-4b52-4318-99db-891b248c07b9.png)

Please read the documentation.
